library/NotificationCenter/MessageDraft/EmailMessageDraft.php

```diff
@@ -170,6 +170,14 @@ public function getHtmlBody()
     }
 
     /**
+     * Returns true if images should not be embedded
+     * @return boolean
+     */
+    public function useExternalImages() {
+        return !!$this->objLanguage->email_external_images;
```

Comment:
Should I attach a new pull request or are you gonna make the changes?
(Sorry for asking dumb questions btw, I'm actually pretty new to contributing to foreign repos. :blush:)